specs/0112-new-contributor-guidelines.md

```diff
@@ -0,0 +1,45 @@
+- Feature Name: Team guidelines for new contributors
+- Created: 2022-05-31
+- Associated Issue: #2305
+
+# Summary
+
+Provide new contributors with guidelines before making their first contribution to a team.
+
+# Motivation
+
+It has been suggested on various occasions and from various sources, including at the localizer workshops, that the onboarding experience for new contributors needs improvements.
+
+Pontoon homepage and tour lead users towards making their first contribution, but don't provide any team-specific information.
+
+On top of that, new users don't get connected with their team, and team managers don't get any explicit information about new contributors joining their teams.
+
+# Feature explanation
+
+## Translate view tooltip
+
+Before users submit their first suggestion to a particular team, a tooltip appears near the text editor, with the following content:
+
+It looks like you haven't contributed to this locale yet.
+
+Make sure to check the <a href="/locale-code/info">team information</a> page before starting, as it might contain important information and language resources.
+
+If you want to contact the team managers, check out the <a href="/locale-code/contributors">team contributors</a> page.
```

Comment:
Would we want to include examples of why someone might want to contact a team manager? If for the purpose of driving engagement/retention of a new potential volunteer, it might be better to phrase this as a call to action. 
Perhaps something like:

`Reach out to your language's team managers from the <a href="/locale-code/contributors">team contributors</a> page if you have questions or want to learn more about contributing.`